### PR TITLE
Lab nicholasc

### PR DESCRIPTION
**Pull request by ncarignan**

### CODE REVIEW

File: lab-nicholasc/__test__/doubly-linked-list.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  });
+  test('remove should delete a node', ()=>{
+    let deletions = new DoublyLinkedList(5);
+    let deletions2 = new DoublyLinkedList(10);
+    let deletions3 = new DoublyLinkedList(15);
+    let deletions4 = new DoublyLinkedList(20);
+    let deletions5 = new DoublyLinkedList(25);
+    let deletions6 = new DoublyLinkedList(30);
+
+    deletions.append(deletions2);
```
Comment (izzybaer) [inline lab-nicholasc/__test__/doubly-linked-list.test.js:22]: shouldn't you be using .remove here?

File: lab-nicholasc/module/topping.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+class Topping{
```
Comment (izzybaer) [inline lab-nicholasc/module/topping.js:3]: what is the purpose of this file? I'm not sure what it's doing

File: lab-nicholasc/__test__/binary-search.test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let binarySearch = require('../lib/binary-search.js');
+
+describe('binary-search.js', () =>{
```
Comment (izzybaer) [inline lab-nicholasc/__test__/binary-search.test.js:5]: you should be testing 2 regular cases and 2 edge cases
